sql/updates/2.17.0--2.16.1.sql

```diff
@@ -0,0 +1,3 @@
+DROP FUNCTION _timescaledb_functions.compressed_data_info(_timescaledb_internal.compressed_data);
+DROP INDEX _timescaledb_catalog.compression_chunk_size_idx;
+DROP FUNCTION IF EXISTS _timescaledb_functions.drop_osm_chunk(REGCLASS);
```

Comment:
The downgrade script should include all from `sql/updates/reverse-dev.sql`.

Reply:
this is the pr into main which will not have the downgrade changes

Reply:
Edited so.